.github/workflows/build.yml

```diff
@@ -176,6 +176,9 @@ jobs:
     name: Verify plugin
     needs: [ build ]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platformType: ['IU', 'WS']
```

Comment:
This is so that we execute the verify step on both IntelliJ Ultimate and Webstorm.  It ensures that the APIs we use in our plugin are compatible with both projects.  I _think_ we might also be able to only depend on Webstorm and assume that IntelliJ Ultimate contains everything from Webstorm and more.

I'm open to changing this PR around to function that way if that is the preferred approach.